packages/openapi-response-validator/index.ts

```diff
@@ -222,7 +223,7 @@ function transformOpenAPIV3Definitions(schema) {
   if (typeof schema !== 'object') {
     return schema;
   }
-  const res = { ...schema };
+  const res = cloneDeep(schema);
```

Comment:
i know this might be ridiculous, but to avoid adding a dependency, would this suffice?

```
const res = JSON.parse(JSON.stringify(schema));
```

Reply:
I went with the lodash function because the JSON stringify/parse method breaks on non json objects like functions or dates, but since these schemas are probably always going to be valid json I'm fine with the JSON thing.

Reply:
i imagine it would be a good thing if it failed due to someone passing in a function here.  would you agree?

Reply:
I do! I would want it to be clear to the user why it failed though. JSON.stringify just erases functions silently so it won't be obvious to the user depending on that function why it isn't there. I think the JSON method is fine for this use case though since these things should 
hopefully get caught earlier.